packages/cx/src/widgets/grid/Grid.scss

```diff
@@ -604,7 +604,7 @@
       bottom: 0;
    }
 
-   .#{$element}#{$name}-col-resizer {
+   %col-resizer {
```

Comment:
`@extend` is deprecated in the next version of Sass. It's safer to add two classes.